components/script/dom/worklet.rs

```diff
@@ -388,9 +373,9 @@ impl WorkletThreadRole {
 #[derive(Clone)]
 struct WorkletThreadInit {
     /// Senders
+    primary_sender: Sender<WorkletData>,
```

Comment:
Not sure what would be added here, other than "the sender to the primary worklet thread" :)